src/Entity/Query/Sql/Query.php

```diff
@@ -30,6 +30,7 @@ public function execute() {
     // revision.
     if (!$original_all_revisions) {
       $this->condition('emr_default_revision', TRUE);
+      $this->sqlQuery->join('entity_meta_default_revision', 'default_revision_table', "base_table.revision_id = default_revision_table.default_revision_id");
```

Comment:
Wonder if we should do something like below where we use `revisionKey` instead of directly putting `revision_id` ?